

\documentclass[11pt]{article}

\title{ Advancing predictive physical modeling through focused development of model systems to drive new modeling innovations}

\usepackage[top=0.5in, bottom=0.5in, left=0.5in, right=0.5in]{geometry}
\usepackage{helvet}
\usepackage{url} % hypderref?
\usepackage{graphicx}
\graphicspath{{figures/}} % The figures are in a figures/ subdirectory.
\renewcommand{\familydefault}{\sfdefault}
\pagestyle{empty}
%\pagestyle{plain}

% Fancy page-width tables
\usepackage{tabularx}

% Use a package for framed boxes
\usepackage{mdframed}

\usepackage[T1]{fontenc}
\usepackage{amssymb}


\usepackage{setspace}
\usepackage{microtype}

\usepackage{amsfonts}
\usepackage{amsmath}

\usepackage{floatrow}

\usepackage[normalem]{ulem} % for nci.bst

\usepackage{sidecap}
\usepackage[abs]{overpic}
\usepackage{wrapfig}


\usepackage{hyperref}
\hypersetup{colorlinks=true, urlcolor=black, citecolor=black, linkcolor=black}

\usepackage[numbers,sort&compress]{natbib} %comment on first run?

\usepackage{multibib}
%\newcites{full,sampl}{A,B}
\newcites{sampl}{Full List of SAMPL References}
%\newbibliography{full}

%\usepackage[round,authoryear]{natbib}
%\usepackage{cite}
%\setlength{\bibsep}{0.00in}


\newcommand{\doi}[1]{\href{http://dx.doi.org/#1}{doi:#1}}

\newcommand{\ac}[1]{{\sc \lowercase{#1}}}

\renewcommand{\baselinestretch}{.93}
%\renewcommand{\baselinestretch}{.90}
\usepackage{wrapfig} 

\usepackage{bibspacing}
\setlength{\bibspacing}{\baselineskip}




\graphicspath{{figs/}}

\makeatletter

\newcommand{\captionfonts}{\footnotesize}

\makeatletter  % Allow the use of @ in command names
\long\def\@makecaption#1#2{%
  \vskip\abovecaptionskip
  \sbox\@tempboxa{{\captionfonts #1: #2}}%
  \ifdim \wd\@tempboxa >\hsize
    {\captionfonts #1. #2\par}
  \else
    \hbox to\hsize{\hfil\box\@tempboxa\hfil}%
  \fi
  \vskip\belowcaptionskip}      
\makeatother

\renewcommand{\figurename}{{\bf Figure}}

% Page numbering.
%\pagestyle{plain}
%\pagenumbering{arabic}

\setlength{\abovecaptionskip}{-5pt}

\makeatother

\renewcommand{\refname}{Bibliography and References Cited}

\setlength{\parindent}{0pt} % Don't indent first line
%\setlength{\parskip}{1ex plus 0.5ex minus 0.2ex} % Add some space between paragraphs
\setlength{\parskip}{0.8ex} % Add some space between paragraphs

\begin{document}

%======================================
% CITE SAMPL REFS FIRST FOR BETTER COMPRESSION
\phantom{
\citesampl{monroe_converging_2014_sampl,muddana_blind_2014_sampl,gallicchio_bedam_2015-1_sampl,mikulskis_free-energy_2014_sampl,hsiao_prediction_2014_sampl,bhakat_resolving_2016_sampl,pal_combined_2016_sampl,yin_sampl5_2016_sampl,bosisio_blinded_2016_sampl,tofoleanu_absolute_2016_sampl,mobley_blind_2014-1_sampl,muddana_sampl4_2014_sampl,sullivan_binding_2016_sampl,deng_distinguishing_2015_sampl,li_testing_2014_sampl,paranahewage_predicting_2016_sampl,klamt_prediction_2016_sampl,tielker_sampl5_2016_sampl,konig_calculating_2016_sampl,luchko_sampl5:_2016_sampl,santos-martins_calculation_2016_sampl,perryman_virtual_2014_sampl,konig_predicting_2014_sampl,voet_combining_2014_sampl,park_extended_2014_sampl,rustenburg_measuring_2016_sampl,reinisch_prediction_2014_sampl,muddana_sampl4_2014-1_sampl,manzoni_prediction_2014_sampl,sandberg_predicting_2014_sampl,brini_adapting_2016_sampl,kamath_prediction_2016_sampl,diaz-rodriguez_predicting_2016_sampl,kenney_prediction_2016_sampl,caldararu_binding_2016_sampl,genheden_all-atom/coarse-grained_2016_sampl,chung_extended_2016_sampl,koziara_testing_2014_sampl,yin_overview_2016_sampl,Bannan:2016:JComputAidedMolDes_sampl,Lee:2016:JComputAidedMolDes_sampl,Jones:2016:JComputAidedMolDes_sampl,Pickard:2016:JComputAidedMolDes_sampl, cao_absolute_2014_sampl,muddana_prediction_2012_sampl,gibb_binding_2013_sampl,klimovich_predicting_2010_sampl,mobley_alchemical_2012_sampl,muddana_sampl3_2012_sampl,skillman_sampl3_2012_sampl,Newman:2011:JComputAidedMolDes_sampl,gallicchio_virtual_2014_sampl,klamt_blind_2010_sampl,fennell_modeling_2011_sampl,ellingson_analysis_2010_sampl,surpateanu_evaluation_2011_sampl,purisima_rapid_2010_sampl,konig_predicting_2011_sampl,kehoe_testing_2011_sampl,kumar_computational_2012_sampl,meunier_predictions_2010_sampl,genheden_extensive_2014_sampl,beckstein_prediction_2014_sampl,coleman_sampl4_2014_sampl,hogues_exhaustive_2014_sampl,reinisch_prediction_2012_sampl,kulp_fragment-based_2012_sampl,klamt_conclusions_2010_sampl,fu_fast_2014_sampl,hamaguchi_force-field_2012_sampl,colas_virtual_2014_sampl,ellingson_efficient_2014_sampl,sulea_predicting_2011_sampl,geballe_sampl2_2010_sampl,ribeiro_prediction_2010_sampl,skillman_sampl2_2010_sampl,gallicchio_prediction_2012_sampl,mikulskis_binding_2011_sampl,geballe_sampl3_2012_sampl,guthrie_sampl4_2014_sampl,nicholls_sampl2_2010_sampl,soteras_performance_2010_sampl,lawrenz_thermodynamic_2012_sampl,sulea_exhaustive_2011_sampl,beckstein_prediction_2011_sampl,benson_prediction_2012_sampl,kast_prediction_2010_sampl, mobley_predictions_2009_sampl,newman_practical_2009_sampl,klamt_prediction_2009_sampl,guthrie_blind_2009_sampl,marenich_performance_2009_sampl,sulea_prediction_2009_sampl,nicholls_samp1_2009_sampl, nicholls_predicting_2008_sampl,chamberlin_performance_2008_sampl, Gosink:2017:J.Phys.Chem.B_sampl, Yang:2017:arXiv:1705.10035[q-bio]_sampl, shirts_lessons_2016_sampl, Bansal:2017:JComputAidedMolDes_sampl}
%Now also add stuff which is cited in the Intro and Aims
\cite{Moult:2014:Proteins, Monastyrskyy:2016:Proteins, Moult:2016:Proteins, Prill:2011:Sci.Signal., Eisenstein:2013:NatBiotech, Saez-Rodriguez:2016:NatRevGenet, Bell:2010:CHANCE, ::XPRIZE, Kay:2011:R&DManage, XPrize:2017:Wikipedia, Mobley:2017:eScholarship, Gathiaka:2016:JComputAidedMolDes}
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% SPECIFIC AIMS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%{\large \bf SPECIFIC AIMS}

%\eject

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% SIGNIFICANCE
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

{\Large \bf SIGNIFICANCE}


\textbf{Physical modeling is poised to transform drug discovery and chemical biology by enabling true molecular design.}
While modeling is used extensively in drug discovery, its main role at present is to aid with idea generation or filter large libraries of compounds for screening. 
Instead, we imagine using computational techniques extensively to guide the design process. 
Consider a medicinal chemist in the not-too-distant future who has just finished synthesizing several new derivatives of an existing inhibitor, and has obtained binding affinity or potency data against the desired biomolecular target. 
Before leaving work, she generates ideas for perhaps 100 new compounds which could be synthesized next, setting her computer to work overnight. 
By morning, the idea compounds have been prioritized based on reliable predictions of their affinity for the desired target, selectivity against antitargets, solubility, and membrane permeability.  
The chemist looks through the predicted properties for the top few compounds, selecting some for synthesis. 
If synthesizing and testing each compound takes several days, this workflow compresses roughly a year's work into a few days.

While this workflow is not yet a reality, significant strides have been made toward accurate binding affinities~\cite{mobley_perspective_2012, christ_accuracy_2014, deng_distinguishing_2015, Sherborne:2016:JComputAidedMolDes, schrodinger_accurate_2015, christ_binding_2016, cui_affinity_2016, verras_free_2016, Aldeghi:2017:J.Am.Chem.Soc.}, solubilities~\cite{Schnieders:2012:J.Chem.TheoryComput., park_absolute_2014, liu_using_2016}, selectivity and drug resistance~\cite{leonis_contribution_2013, Aldeghi:2017:J.Am.Chem.Soc.}, and membrane permeability~\cite{lee_permeability_2016, comer_permeability_2014}. 
A considerable amount of science and engineering still remains to make this vision a reality.
However, given recent progress, the question now seems more one of \emph{when} rather than \emph{whether}.

Widespread availability of inexpensive graphics processing units (GPUs) provides a 100-fold increase in price-to-performance ratio over CPUs, while advances in automation~\cite{liu_lead_2013} and sampling protocols have helped simulation-based techniques reach the point where they now begin to be genuinely useful in guiding drug discovery for a limited \emph{domain of applicability}~\cite{mikulskis_large-scale_2014, homeyer_binding_2014, Sherborne:2016:JComputAidedMolDes,  schrodinger_accurate_2015, christ_binding_2016, cui_affinity_2016, verras_free_2016}.
Specifically, in some situations, free energy calculations appear to be capable of achieving RMS errors of 1--2 kcal/mol with current force fields, even in prospective applications, sufficient to drastically reduce the number of molecules that must be synthesized and assayed~\cite{shirts_free-energy_2010, Aldeghi:2017:J.Am.Chem.Soc., schrodinger_accurate_2015}.
As a consequence, pharmaceutical companies are beginning to use these methods in active discovery projects~\cite{Sherborne:2016:JComputAidedMolDes}.

\textbf{Despite progress, current modeling methods suffer from severe limitations hindering their widespread use in molecular design.}
For example, even ``small'' protein conformational changes not gracefully handled by current methodologies can yield errors up to 5 kcal/mol in calculated binding free energies~\cite{lim_sensitivity_2016}, force field limitations still pose major challenges~\cite{rocklin_blind_2013}, and the inability to treat important chemical effects like protonation and tautomer equilibria drastically limits the domain of applicability.
{\bf For many pharmaceutically relevant systems, the most important sources of error---and modeling challenges---are not yet clear.~\cite{Sherborne:2016:JComputAidedMolDes}}

These challenges frustrate progress in the field. 
Method developers focus on addressing some specific challenges, but {\bf new methods are tested on disparate data sets making it difficult to gauge progress and identify valuable methods when they are developed}.
And applications projects tend not to take advantage of the latest methods -- in part because their value is not yet clear. 
Thus valuable methods can languish relatively unused in the literature for many years without impacting applications in drug discovery.

{\bf Here, we propose a crowdsourcing, challenge-based approach to leverage and drive methodological innovation, accelerating progress.} 
Specifically, we propose a series of community blind prediction challenges focused on pushing the limits of predictive techniques, providing a bridge between challenging but tractable problems and pharmacologically relevant but currently intractable problems.
As discussed below, these challenges take advantage of proven crowdsourcing-based approaches to science, allowing head-to-head comparisons of existing methods and fostering development of new methods.
With regular cycles of challenges and learning, we provide an environment for rapid innovation from ``unconventional innovators''~\cite{Kay:2011:R&DManage} from diverse fields who form teams, appling and integrate diverse approaches~\cite{Saez-Rodriguez:2016:NatRevGenet}.
Regular direct comparisons of methods allow progress to be quickly recognized and propagate across the field.

{\bf A key aspect of these challenges is that they must be prospective, bringing new data to serve as the basis for each challenge.}
Without new data, there can be no blind prediction challenge, so a key focus of this proposal is generating new data in order to run these challenges.
But generating new data provides additional benefits over utilizing existing data in retrospective tests.
First, we are able to ensure high data quality.
Additionally, retrospective tests cannot provide an accurate measure of the utility of methods in prospective tests. 
Retrospective tests can also easily result in over-fitting, where researchers apply a variety of protocols until apparently significant results are obtained by chance~\cite{Nuzzo:2015:Nature}.
Retrospective work also may utilize prior knowledge, even if unintentionally.  
For example, if the binding mode of a ligand is already known crystallographically, a researcher may use that binding mode in retrospective tests, whereas prospective or design work would require first selecting among candidate binding modes, introducing substantial uncertainty unaccounted for in the retrospective statistics~\cite{mobley_predicting_2007, boyce_predicting_2009, mobley_perspective_2012}.
While one might argue that discovery projects provide an opportunity for prospective tests, they are not adequate -- partly because often, the predicted compounds are in fact never tested~\cite{christ_binding_2016} or the experimental data necessary to assess the quality of the predictions is absent---for example, because binding affinities are not measured or no crystallography is available. 
Additionally, the data in such cases is usually unavailable to the community until it is published, meaning that for most, these become retrospective tests.

%Capital Challenge?
{\bf A successful crowdsourcing Challenge must be tractable -- not too easy but not too hard -- as unsolvable Challenges fail to drive progress}~\cite{Saez-Rodriguez:2016:NatRevGenet}. 
This means that we not only must generate high quality experimental data for prospective tests, but our Challenges must be carefully designed to maximize community learning, pushing the limits of today's technology.
While the Drug Design Data Resource (D3R~\cite{Gathiaka:2016:JComputAidedMolDes}, discussed further below) provides an existing community blind challenge on protein-ligand binding, it focuses on using pre-existing pharmaceutical datasets, rather than on measuring new data carefully selected maximize community learning~\cite{Gathiaka:2016:JComputAidedMolDes}. 
D3R serves well to assess where we are now---but because it relies on existing donated data, D3R challenges can prove nearly intractable, resulting in only modest new insight~\cite{D3R_lessons_learned}.

\textbf{Modeling accuracy advances most rapidly when progress toward a complex goal can be decomposed into resolving a series of tractable problems and assessed using high quality data~\cite{Saez-Rodriguez:2016:NatRevGenet}.}
To make rapid progress, our field needs an effort which focuses on exactly this decomposition -- solving specific \emph{component} problems of the overall problem of interest via collection and curation data that highlights these problems, and drives progress via prospective Challenges.
This process allows the entire community to learn from both methodological success and failure. 

Much as unit testing is an indispensible tool for discovering where bugs in a program are hiding when complex integration tests fail, exercises like SAMPL are valuable in pinpointing and correcting modeling errors.
Projects like D3R allow us to see that performance falls short in pharmaceutical applications, but SAMPL focuses on highlighting specific points of failure and driving improvement in those areas. 
Thus, SAMPL Challenges focus on properties that attempt to isolate likely sources of error, such as physical properties of small molecules (hydration free energies, aqueous tautomer ratios, partition or distribution coefficients between aqueous and nonpolar phases) as well as small molecule binding to targets of reduced complexity (such as host-guest binding, and binding of fragments to trypsin and HIV integrase).

Our {\bf Statistical Assessment of Modeling of Proteins and Ligands (SAMPL) series of challenges has already proven it drives dramatic improvements of modeling}, as have larger-scale challenges elsewhere in modeling~\cite{Saez-Rodriguez:2016:NatRevGenet}.
SAMPL, born out of frustration with the lack of venues for comparing predictive accuracy on a level playing field, was initiated by Anthony Nicholls of OpenEye software in 2007/2008~\cite{nicholls_predicting_2008}, and has run challenges approximately every two years since then~\cite{nicholls_samp1_2009, mobley_predictions_2009, geballe_sampl2_2010, geballe_sampl3_2012, mobley_blind_2014-1, muddana_sampl4_2014, Bannan:2016:JComputAidedMolDes, yin_overview_2016}.
Governance transitioned to an unfunded academic collaboration during SAMPL3 in 2012; this collaboration ran SAMPL4 (2014), SAMPL5 (2016), currently SAMPL6 (2017-2018). 
The PI of this proposal (Mobley) was a primary organizer of SAMPL4-6 (2014--present).
{\bf SAMPL has already been a tremendous community resource, resulting in some 100 publications which are typically cited 5--50 times or more each}~\citesampl{monroe_converging_2014_sampl,muddana_blind_2014_sampl,gallicchio_bedam_2015-1_sampl,mikulskis_free-energy_2014_sampl,hsiao_prediction_2014_sampl,bhakat_resolving_2016_sampl,pal_combined_2016_sampl,yin_sampl5_2016_sampl,bosisio_blinded_2016_sampl,tofoleanu_absolute_2016_sampl,mobley_blind_2014-1_sampl,muddana_sampl4_2014_sampl,sullivan_binding_2016_sampl,deng_distinguishing_2015_sampl,li_testing_2014_sampl,paranahewage_predicting_2016_sampl,klamt_prediction_2016_sampl,tielker_sampl5_2016_sampl,konig_calculating_2016_sampl,luchko_sampl5:_2016_sampl,santos-martins_calculation_2016_sampl,perryman_virtual_2014_sampl,konig_predicting_2014_sampl,voet_combining_2014_sampl,park_extended_2014_sampl,rustenburg_measuring_2016_sampl,reinisch_prediction_2014_sampl,muddana_sampl4_2014-1_sampl,manzoni_prediction_2014_sampl,sandberg_predicting_2014_sampl,brini_adapting_2016_sampl,kamath_prediction_2016_sampl,diaz-rodriguez_predicting_2016_sampl,kenney_prediction_2016_sampl,caldararu_binding_2016_sampl,genheden_all-atom/coarse-grained_2016_sampl,chung_extended_2016_sampl,koziara_testing_2014_sampl,yin_overview_2016_sampl,Bannan:2016:JComputAidedMolDes_sampl,Lee:2016:JComputAidedMolDes_sampl,Jones:2016:JComputAidedMolDes_sampl,Pickard:2016:JComputAidedMolDes_sampl, cao_absolute_2014_sampl,muddana_prediction_2012_sampl,gibb_binding_2013_sampl,klimovich_predicting_2010_sampl,mobley_alchemical_2012_sampl,muddana_sampl3_2012_sampl,skillman_sampl3_2012_sampl,Newman:2011:JComputAidedMolDes_sampl,gallicchio_virtual_2014_sampl,klamt_blind_2010_sampl,fennell_modeling_2011_sampl,ellingson_analysis_2010_sampl,surpateanu_evaluation_2011_sampl,purisima_rapid_2010_sampl,konig_predicting_2011_sampl,kehoe_testing_2011_sampl,kumar_computational_2012_sampl,meunier_predictions_2010_sampl,genheden_extensive_2014_sampl,beckstein_prediction_2014_sampl,coleman_sampl4_2014_sampl,hogues_exhaustive_2014_sampl,reinisch_prediction_2012_sampl,kulp_fragment-based_2012_sampl,klamt_conclusions_2010_sampl,fu_fast_2014_sampl,hamaguchi_force-field_2012_sampl,colas_virtual_2014_sampl,ellingson_efficient_2014_sampl,sulea_predicting_2011_sampl,geballe_sampl2_2010_sampl,ribeiro_prediction_2010_sampl,skillman_sampl2_2010_sampl,gallicchio_prediction_2012_sampl,mikulskis_binding_2011_sampl,geballe_sampl3_2012_sampl,guthrie_sampl4_2014_sampl,nicholls_sampl2_2010_sampl,soteras_performance_2010_sampl,lawrenz_thermodynamic_2012_sampl,sulea_exhaustive_2011_sampl,beckstein_prediction_2011_sampl,benson_prediction_2012_sampl,kast_prediction_2010_sampl, mobley_predictions_2009_sampl,newman_practical_2009_sampl,klamt_prediction_2009_sampl,guthrie_blind_2009_sampl,marenich_performance_2009_sampl,sulea_prediction_2009_sampl,nicholls_samp1_2009_sampl,nicholls_predicting_2008_sampl,chamberlin_performance_2008_sampl, Gosink:2017:J.Phys.Chem.B_sampl, Yang:2017:arXiv:1705.10035[q-bio]_sampl, shirts_lessons_2016_sampl, Bansal:2017:JComputAidedMolDes_sampl}.
SAMPL data sets become so valuable to the community that they foster retrospective tests long after the challenges have ended (e.g.~\cite{Yang:2017:arXiv:1705.10035[q-bio], Gosink:2017:J.Phys.Chem.B, Harger::J.Comput.Chem., Bradshaw:2016:J.Chem.TheoryComput.}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%SAMPL HISTORY
\begin{wrapfigure}[20]{r}{7cm}
\vspace{-0.15in}
\begin{centering}
\includegraphics[width=\textwidth]{figures/history_v1.pdf}

\end{centering}
\footnotesize
\caption{\label{figure:sampl_history}  
\textbf{SAMPL historical participation~\cite{mobley_blind_2014-1}.} 
Historical participation in SAMPL host-guest + solvation/distribution challenges has climbed rapidly, and we expect this trend to continue. The number of participating groups is shown in blue, and the number of submissions in green. The inset shows the diversity of methods employed for the SAMPL4 hydration challenge, which is typical for SAMPL. 
While the overall number of participants appears lower in SAMPL5, this is primarily because SAMPL4 included an HIV integrase binding prediction challenge, whereas such datasets are (as of SAMPL5) now handled by D3R.
Despite this shift, the number of submissions in SAMPL5 was the highest yet.
}
\end{wrapfigure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Here, we \textbf{design a new series of SAMPL Challenges specifically to guide the improvement of models}.
Until now, this has been impossible, because SAMPL's existence has required ``donation'' of data and time, giving us no ability to gather datasets tailored to our purpose. 
Despite these past limitations, our recent survey reveals that SAMPL has had a significant effect on the field and past participants are overwhelmingly enthusiastic~\cite{Mobley:2017:eScholarship}, including about the plans proposed here. 
One of our most frequent requests is for larger datasets, which we propose here. 
Our proposed Challenges bridge the gap between calculations of simple physical properties that isolate forcefield inaccuracies from sampling challenges, like hydration---which can already be calculated fairly accurately~\cite{mobley_blind_2014-1}---and the D3R Grand Challenges on protein-ligand binding, which are a major source of consternation for the community~\cite{ignjatovic_binding-affinity_2016, deng_large_2016, sunseri_d3r_2016, Gathiaka:2016:JComputAidedMolDes}.
Unless this gap is bridged, there is the very real possibility that modeling may simply continue to fall far short of expectations in pharmaceutical challenges like D3R for reasons which are unclear.


\textbf{Our major goal is to rapidly advance predictive modeling to where it can guide biomolecular design, and extension of the SAMPL challenges will do exactly that.} This work will play a vital role in enhancing the work being done on \emph{existing} data by D3R, helping prepare methods for application to the more challenging systems emerging from pharma in D3R's challenges. 

{\Large \bf INNOVATION}

This work proposes the SAMPL challenge as an \emph{innovation engine} for molecular design.
Specifically, collaborative scientific competitions or Challenges are a proven model for inducing innovation in a given field~\cite{Saez-Rodriguez:2016:NatRevGenet, Kay:2011:R&DManage, Eisenstein:2013:NatBiotech} providing a crowdsourced way of finding and identifying robust new methodologies~\cite{Saez-Rodriguez:2016:NatRevGenet}. 
The proposed series of SAMPL challenges brings crowdsourcing to bear on molecular interactions in a way which focuses on solving specific, incremental problems that impair our ability to accurately and robustly predict these interactions.

Thus, while aspects of the proposed work are indeed innovative (as we argue below), the most important innovation will come from the work of participants in SAMPL Challenges, as is the case in all crowdsourcing approaches to innovation~\cite{Saez-Rodriguez:2016:NatRevGenet, Kay:2011:R&DManage}.
This reliance on others for innovation in fact is a strength of this model -- it is not always obvious where the most important innovations may come from, but Challenges draw together teams of diverse entrants, including unconventional innovators who may apply unorthodox, risky, or radical technologies~\cite{Kay:2011:R&DManage, Saez-Rodriguez:2016:NatRevGenet,}, resulting in progress both from the usual suspects and from unexpected sources.
This means it is not necessary to decide in advance who is best positioned to solve the key problems~\cite{Saez-Rodriguez:2016:NatRevGenet,} -- the Challenge itself drives innovation and allows it to be recognized, regardless of the source.

Crowdsourcing models for innovation have an ample track record, with the XPrize driving major headlines~\cite{::XPRIZE, Kay:2011:R&DManage, XPrize:2017:Wikipedia}, and the Netflix Prize also familiar to many~\cite{Bell:2010:CHANCE}. 
While there are other Challenges in the area of biomolecular modeling, such as D3R~\cite{Gathiaka:2016:JComputAidedMolDes}, the p$K_a$ cooperative~\cite{Nielsen:2011:Proteins}, CAPRI~\cite{Janin:2005:ProteinScience} and CASP~\cite{Moult:2014:Proteins},
\textbf{no other blind challenge focuses specifically on data tailored and collected to drive quantitative protein-ligand modeling}.

SAMPL is designed in the spirit of the DREAM series of Challenges, an extremely successful series of challenges in the biomedical area~\cite{Prill:2011:Sci.Signal., Eisenstein:2013:NatBiotech, Saez-Rodriguez:2016:NatRevGenet} which had more than 800 participants in its most recent iteration. 
As recommended by the DREAM challenge organizers~\cite{Saez-Rodriguez:2016:NatRevGenet}, SAMPL focuses on high quality data, decomposing complex problems into component problems which are themselves tractable but challenging in order to benefit from crowdsourcing, and drawing together diverse teams and methods to allow various solutions to be tested and integrated.

To a large extent, SAMPL already follows this model, and has already been important in advancing protein-ligand modeling, as evidenced by the results of our survey of the field~\cite{Mobley:2017:eScholarship}.
We also attach testimonials highlighting some specific ways SAMPL has advanced science in our field.
Several historical examples further serve to highlight how SAMPL can foster innovation (see also Figure~\ref{figure:sampl5_logD} and our SAMPL bibliography). 
The first several SAMPL challenges on hydration free energies had rather hit-and-miss performance, highlighting pitfalls of existing methods and force fields which led to marked improvements in PB models~\cite{nicholls_samp1_2009, ellingson_analysis_2010,ellingson_efficient_2014}, recognition of some limitations of fixed-charge force fields ~\cite{mobley_alchemical_2012, Fennell:2014:J.Phys.Chem.B},
repair of some of these force field deficiencies~\cite{mobley_alchemical_2012, Fennell:2014:J.Phys.Chem.B, paranahewage_predicting_2016},
and helped motivate alternate implicit or hybrid solvent models~\cite{sulea_predicting_2011, li_testing_2014, brini_adapting_2016}.
Shifts in protonation state and tautomer proved particularly important in SAMPL5's log~D challenge~\cite{Bannan:2016:JComputAidedMolDes, klamt_prediction_2016}, as they will in binding.
Host-guest binding Challenges~\cite{mobley_predicting_2016},
have highlighted the importance of salt effects~\cite{yin_overview_2016, muddana_blind_2014, mobley_predicting_2016}
and in some cases revealing severe force field limitations~\cite{yin_sampl5_2016, muddana_sampl4_2014-1}, pointing the way forward for improving predictive models of molecular interactions~\cite{yin_toward_2015, mobley_predicting_2016}.

Thus, this work designs a new series of SAMPL Challenges tailored to progressively advance computational methods for binding (Aim 4), gathering the requisite experimental data to drive these challenges (Aims 1-3). 


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% FIGURE: TIMELINE
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[h]
\vspace{-0.10in}
\begin{centering}
\includegraphics[width=\textwidth]{figures/Timeline3_cropped.pdf}
\end{centering}
\vspace{-0.25in}
\caption{\footnotesize {\bf Timeline for our activities.} Activities covered by this grant include data collection and SAMPL challenges on our three major components (physical properties, host-guest binding, and protein-ligand binding), with each challenge cycle color-coded separately.  
Data collection within each Aim is shown by a colored bar indicating what is measured and curated.
Data collection/curation is followed by a submission window for that challenge component, then all results and analysis are returned to participants and posted on the SAMPL website; this also will nucleate more detailed long-term discussion on the relevant Slack channel.
At this point, we will also release the data to the public as a high quality benchmark.
Each component will then wrap up with a virtual meeting focused on lessons learned and areas which need further exploration; these will be recorded and posted on our website to assist in rapid dissemination of new insights. 
Virtual meetings precede the submission window for the next SAMPL challenge, giving the opportunity to incorporate lessons learned for the next challenge.
Submission windows and virtual meetings are staggered across categories so that participants can be involved in all three major areas without multiple simultaneous deadlines.
In-person meetings are co-hosted with D3R and will occur every two years, supplemented by effort-wide virtual meetings in between.
Special issues of JCAMD will have deadlines shortly after the virtual meeting on the protein-ligand challenge for that year, and a 4-5 month timeline (based on historical experience) from the submission deadline until the special issue appears (with the first papers appearing online substantially sooner).
Rapid dissemination of insights is critical for rapid progress, so we highly encourage the use of preprints and informal reports to supplement the special issue.
%UPDATE TIMELINE. 
\vspace{-0.1in}
\label{figure:timeline}}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

While the greatest innovation here is the new science that comes out the proposed Challenges, we also innovate on several other fronts.
Aim 3 includes a development of an innovative informatics platform to facilitate the rapid identification and study of particularly informative protein-ligand systems that are both experimentally tractable for high-throughput biophysical measurements and focus on specific challenges of interest.
And an innovative, automated wetlab helps produce the high quality data measured in Aim 3. 
Additionally, students in our computational groups perform reference calculations to test the accuracy of current state-of-the-art techniques.
These help provide training for students, but also drive innovation by (1) Benchmarking the latest method developments against current ``best practices'' methods (by doing calculations via both approaches); (2) Facilitating learning, allowing others to compare against our results to determine how a change in method or force field impacts results; (3) Focusing the field on key issues by doing sensitivity analysis to whether conditions such as ionic strength, protonation state, tautomer choice, etc., impact computed values.

{\Large\textbf{APPROACH}}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%SAMPL DC FIGURE
\begin{wrapfigure}[22]{r}[0pt]{9cm}
%\vspace{-0.25in}
\begin{centering}
\includegraphics[width=\textwidth]{figures/sampl5_logD_v1.pdf}

\end{centering}
\footnotesize
\caption{\label{figure:sampl5_logD}  
\textbf{Lessons learned from SAMPL5 log~D predictions.} 
Predictions of log~D values for SAMPL5 provided a number of key lessons. (A) Methods which treated multiple protonation and tautomeric states in their predictions performed dramatically better than those which did not; here, red dots move to x symbols when these effects are treated, improving accuracy in every case~\cite{Pickard:2016:JComputAidedMolDes}. (B) Re-parameterization of a force field to more accurately reproduce pure solvent dielectric constants resulted in dramatically better predictions (top) than the original force field (bottom)~\cite{paranahewage_predicting_2016}. 
}
\end{wrapfigure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

A series of data-driven Challenges will drive systematic advances in modeling for biomolecular design.
To conduct these, we collect carefully targeted experimental datasets focused around key design challenges that cover a spectrum of complexity (Aims 1--3), and use these to field SAMPL Challenges driving innovation (Aim 4), with each Challenge including components from each of Aims 1--3.
Our Challenges help isolate and address limitations in quantitative physical modeling, using a competitive, crowdsourcing model to encourage and evaluate multiple solutions from the community.
Data collection bring togethers multiple laboratories and industrial collaborators, including both theorists and experimentalists: graduate students from the Mobley and Chodera laboratories are paired with well-equipped experimental groups in industry to collect physical property data (Aim 1); Gibb and Isaacs, leading experimentalists in supramolecular chemistry, work with theorist Mobley to perform host-guest affinity measurements (Aim 2); and the Chodera lab applies new automated approaches to identify suitable protein-ligand systems and measure binding (Aim 3).


\textbf{\underline{Aim 1: Collect new physical property datasets to assess accuracy and spur improvements in force fields}}
\textbf{\underline{and modeling of protonation states and tautomers.}} 


Simple physical properties such as solvation, partitioning, and protonation equilibria can be calculated quite precisely (but not necessarily accurately) with physical methods, allowing quantitative comparison between calculations and experiment and revealing and isolating deficiencies in our models. 
These properties allow us to directly probe force field accuracy and chemical effects like protonation and tautomer handling in the absence of slow conformational changes and other effects which complicate assessment in protein-ligand systems. 

{\bf Rationale:}
We will generate new solution-phase physical property measurements for drug-like molecules to motivate improvements in force fields and handling of protonation states and tautomers.
This builds on our work on water-cyclohexane distribution coefficients for SAMPL5 (in partnership with Genentech), which revealed major issues with handling of protonation states and tautomers~\cite{Bannan:2016:JComputAidedMolDes}  as well as serious forcefield limitations~\cite{paranahewage_predicting_2016} (Figure~\ref{figure:sampl5_logD}).
Distribution coefficients give the equilibrium ratio of concentrations of a solute between aqueous and nonpolar phases, and thus relate to transfer free energies from aqueous to more protein- or membrane-like environments.
Thus, they capture many of the characteristics of transfer of drugs from water into binding sites but absent challenges with receptor conformational sampling and specific ligand-receptor interactions.
In SAMPL5, distribution coefficients were challenging enough that many methods performed poorly, with even the best methods having accuracies less than would be expected based on hydration free energies in water~\cite{Bannan:2016:JComputAidedMolDes}, yet failures were informative and the major sources of error were issues which will also plague prediction of ligand-receptor interactions.
In some respects, distribution coefficients posed the ideal SAMPL challenge, hitting the sweet spot in terms of difficulty---difficult enough that clear failures were frequent, with ample room for improvement, but not so difficult that the reasons for failure were unclear in general. 
Still, many models consistently disagreed with experiment for some compounds~\cite{paranahewage_predicting_2016, klamt_prediction_2016, Bannan:2016:JComputAidedMolDes, rustenburg_measuring_2016}, revealing the impact that targeted follow-up experiments (such as those we will conduct here) could have on improving models.

\textbf{Targeted Challenges focused on solution-phase properties provide a mechanism to break complex problems into challenging but tractable components}.  
Previous SAMPL challenges in the area of solution-phase properties have already driven significant progress, but a frequent request is for more data and for follow-up experiments to check questionable experimental values~\cite{Mobley:2017:eScholarship}. 
With our previous reliance on donated data, this was impossible, doing our own data collection, we can expand the size and quality of Challenge data sets, allowing us to do things like ensure the full dynamic range of log~D values is covered, unlike in SAMPL5~\cite{rustenburg_measuring_2016, Bannan:2016:JComputAidedMolDes}. 
Follow-up experiments will also become routine. 
This will improve our ability to learn from the data---for example, SAMPL5's lack of dynamic range meant that, when calculated values often spanned a larger dynamic range than the experimental values, it was unclear if this was an artifact of the data set itself, experimental limitations, or force field problems~\cite{rustenburg_measuring_2016, Bannan:2016:JComputAidedMolDes, paranahewage_predicting_2016, klamt_prediction_2016}.  

Solution-phase data and Challenges will also improve lessons learned from binding Challenges using data of Aims 2 and 3.
Specifically, ligands being studied in host-guest and protein-ligand binding studies will be prioritized for inclusion in solution-phase datasets; if solution-phase data is already public for these, analogs or fragments will be included. 
Thus, for example, if a binding free energy proves especially difficult to predict, researchers will be able to see if they also did poorly on predicting its solution-phase properties such as protonation states and tautomers, thereby focusing on the origin of the error.

Below, we summarize plans for data collection for the SAMPL7-11 challenges of Aim 4 (see also Figure~\ref{figure:timeline}).
These specific data sets were selected based on a desire to help the field resolve the problems encountered in SAMPL5 then progress to accurate estimation of new properties.
Data sets will be larger than prior SAMPL challenges --- at least 96 compounds for good statistics ---  though when possible, a much larger amount of data will be collected.

\begin{wraptable}[18]{r}{6.5cm}
\vspace{-0.2in}
\footnotesize
\begin{tabular}{l | l}
{\bf drug} & {\bf features} \\
\hline
memantine & adamantane; 1:1 \\
saxagliptin & adamantane; 1:1 \\
premarin & steroid \\
pancuronium & steroid\\
varenicline & 1:1 vs 1:2 \\
valsartan & p$K_a$ 4.37 \\ 
omeprazole & p$K_a$ 4.77 \\
ranolazine & p$K_a$ 7.17; epitopes \\
pradaxa & p$K_a$ 3.87; epitopes \\
nilotinib & epitopes; p$K_a$ 6.3 \\
sensipar & epitopes; folding \\
vyvance & diamine; epitopes; folding \\
minocycline & tetracyclin; amino aniline \\
\end{tabular}
\caption{\textbf{\label{table:CB} Selected drugs whose binding to CB[$n$] hosts will be assayed for SAMPL6, 8, and 10 challenges (SA 2.1).}  %SAMPL7?
These drugs bind to the cucubituril-based host systems considered here, some at high affinity, so measuring their affinities provides a way to test methods for predicting binding interactions absent complexities present in protein-ligand systems.
}
\vspace{-0.1in}
\end{wraptable}

\textbf{SAMPL7: Cyclohexane/water and octanol/water distribution coefficients, given pKa.}
Building on the success of distribution coefficient measurements in SAMPL5 and their surprising ability to motivate rapid advances in physical modeling methodologies~\cite{Bannan:2016:JComputAidedMolDes}, we will measure cyclohexane-water distribution coefficients at pH 7.4 for a new batch of commercially-available drug- and fragment-like molecules overlapping with compounds being studied in Aims 2 and 3.
Given the routine nature of octanol-water distribution coefficient measurements and indications that their prediction may be computationally tractable~\cite{Bhatnagar:2013:PhysicalChemistryChemicalPhysics, bannan_calculating_2016} despite the heterogeneous structure of the wet octanol phase~\cite{Kollman:1996:AccountsofChemicalResearch}, we will also measure octanol-water distribution coefficients for the same compounds.
Because p$K_a$ prediction was difficult but critical for SAMPL5, we will focus SAMPL7 on forcefields and tautomers by measuring and providing p$K_a$ values, revisiting p$K_a$ prediction in SAMPL8 and 9.
A similar approach is currently being applied for SAMPL6 (data collection is in process), so depending on the outcomes of the SAMPL6 Challenge, plans here may be adjusted modestly.

\textbf{SAMPL8 and SAMPL9: Distribution coefficients and pKa's for drug-like molecules.} 
While much less complex than protein-ligand affinities, distribution coefficient measurements still conflate the challenging issues of protonation state and tautomer prediction, as well as transfer into different environments which may contain small but important quantities of co-solvents. 
Thus, we will separate these issues to allow treatment of protonation and solvation equilibria separately. 
Thus for SAMPL8-9, then, we will measure p$K_a$ values for an extensive set of drug-like molecules in water as well as distribution from water into non-water solvents.
SAMPL8 will focus on octanol-water and cyclohexane-water, and SAMPL9 will shift to alternate solvents to ensure that models are capable of handling a variety of non-water phases. 
As in SAMPL7, we plan to focus partly on molecules and fragments overlapping with compounds being studied in Aims 2-3.

\textbf{SAMPL10-11: Solubility prediction and membrane permeability.}
With solubility predictions now becoming tractable~\cite{Schnieders:2012:J.Chem.TheoryComput., park_absolute_2014, liu_using_2016} (with Schr\"{o}dinger also working on amorphous solubility prediction), solubility measurements will be a valuable test for SAMPL10, combining the solvation aspects of SAMPL1-8 with a new solid phase component.
New computational techniques are targeting membrane permeability~\cite{lee_permeability_2016, comer_permeability_2014}, and this is experimentally accessible (see support letters from Pfizer and Merck), leading to our interest in permeability for SAMPL11.
However, it is likely that both solubility and permeability will still be extremely challenging -- so we plan to also measure distribution coefficients and pKa's for the same compounds to help participants isolate potential points of failure to the solution- or solid-phase portions of their calculations.

%\eject
  
{\bf Experimental plan:}
Experimental data will be collected in collaboration with our pharma partners, roughly following the model used for SAMPL5, where Chodera lab student Bas Rustenburg adapted a Genentech workflow to conduct cyclohexane-water log~D measurements at Genentech~\cite{rustenburg_measuring_2016}.
A similar approach is currently being employed with Merck for SAMPL6.
Thus, Mobley and Chodera lab students will visit industry partners collecting targeted datasets.
Working with industry collaborators (see Letters of Collaboration from Genentech, Pfizer, GSK, and Merck) gives us substantial access to equipment and high-throughput measurement workflows---such as the Sirius T3 from Sirius Analytical (which can measure partition/distribution coefficients, p$K_a$s, and solubilities for molecules with titratable groups)---automation equipment, and compound libraries---for the purposes of rapidly collecting targeted datasets.
This model has been proven to work~\cite{rustenburg_measuring_2016}, and our partners see the value of this data and SAMPL to the modeling community.

Overall, Aim 1 extends prior SAMPL Challenges via data focused on quantitative prediction of physical properties of tremendous relevance to accurately predicting biomolecular interactions, paving the way to applications in more complex systems addressed in Aims 2 and 3.


\textbf{\underline{Aim 2: Measure affinities of drug-like compounds in supramolecular hosts to challenge quantitative}}
\textbf{\underline{models of binding in systems not plagued by major receptor sampling issues.}}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%SAMPL HG FIGURE
\begin{wrapfigure}[23]{l}[0pt]{7cm}
\vspace{-0.23in}
\begin{centering}
\includegraphics[width=\textwidth]{figures/sampl3_and_sampl5.pdf}

\end{centering}
\footnotesize
\caption{\label{figure:hg_sampl}  
\textbf{The best host-guest binding predictions of SAMPL3~\cite{muddana_sampl3_2012} and SAMPL5~\cite{yin_sampl5_2016}.} 
Binding free energy predictions have shown clear improvements from SAMPL3 to SAMPL5 as the major challenges become understood and are treated better by models, though a systematic offset remains in the best SAMPL5 predictions (yellow). Dashed lines denote errors of $\pm1.5$ kcal/mol.
}
\end{wrapfigure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%Try to recast a bit in terms of grand challenge of binding prediction, SpaceX-ish stuff.
Aim 1 focuses on the behavior of small molecules and its environment-dependence, in the absence of receptors and the associated potential for slow sampling, strong specific interactions, and other challenges such as salt effects.
Binding in host-guest systems retains many of the same challenges seen in Aim 1 and introduces strong specific interactions and other challenges like salt effects~\cite{mobley_predicting_2016}, while still avoiding many of the issues with slow sampling (of protein conformational changes, ions, and ligand binding modes) seen in protein-ligand interactions.
That is, binding in host-guest systems introduces a wider variety of challenges relevant to biomolecular interactions, but without the full array of challenges seen in protein-ligand interactions, as reviewed recently~\cite{mobley_predicting_2016}.
Thus, new data for SAMPL challenges on host-guest binding is critical to provide challenges of intermediate complexity between those of physical properties and those in biomolecular binding.
We believe that host-guest binding challenges provide a vital step towards accurately modeling biomolecular interactions, focusing the field on issues not commonly encountered in physical property challenges (such as the importance of accurately modeling ionic conditions) that are highly relevant for protein-ligand interactions.
Already, over the past several SAMPL challenges, host-guest systems have provided key tests for modeling of binding interactions, resulting in new attention paid to how co-solvents and ions modulate binding (resulting in errors of up to 5 kcal/mol when these effects are neglected) and the importance of adequately sampling water rearrangements~\cite{muddana_sampl4_2014, mobley_predicting_2016, yin_overview_2016, bhakat_resolving_2016}. 

This new attention has resulted in clear improvements as participants begin to treat the relevant effects more accurately (Figure~\ref{figure:hg_sampl}).
Host-guest binding proves remarkably difficult to model accurately~\cite{henriksen_computational_2015}, in part due to force field limitations (resulting in new force field work~\cite{yin_toward_2015}).


%Update this section to shift numbering and tangentially reference SAMPL6.
Here, we design a series of SAMPL challenges focused on two classes of host-guest systems---cucurbiturils and analogs (SA 2.1) and Gibb's deep-cavity cavitands (GDCCs, SA 2.2)---both of which build on prior SAMPL challenges.
These two sets of systems exhibit different challenges as recently reviewed~\cite{mobley_predicting_2016}, with the hosts of 2.1 bringing relatively modest co-solvent and ion effects but some receptor sampling problems for the acyclic hosts, and the GDCCs of 2.2 bringing profound ion and co-solvent effects as well as water sampling challenges.
Methods which perform well on one class may not perform well on the other~\cite{mobley_predicting_2016}, since the distinct sets of challenges highlight different limitations.
This diversity drives more innovation than would a focus on a single host class.


%Chodera suggested this on the octa-acid section:
%* We need to emphasize why the octa-acid system has proven to be so valuable for modelers in the introduction: What specific physical challenges does the system probe, and what have we learned? We also need to add citations to the meta-analysis and maybe even all of the papers that addressed this system. David can likely address this.
%DLM: It's tempting to add material along these lines to the beginning of both the octa-acid and CB sections (i.e. making each start with a "history" section), and I certainly have material on that, but given that we're a bit over a page too long before I even add figures, I'll leave this as a placeholder in case we somehow find ourselves with space to add it. We can't afford the space at present unless we find something to shrink dramatically.


\textbf{Subaim 2.1: Cucurbituril-based receptors as model binding systems}
%Isaacs science

\begin{figure}[h]
\begin{centering}
\resizebox{\textwidth}{!}{\includegraphics{figures/CB.pdf}}

\end{centering}

\caption{
\label{figure:CB} \footnotesize {\bf SAMPL6-10 host-guest challenges will feature cucubituril hosts and analogs, including Me$_4$CB[8], glycoluril hexamer, and acyclic CB[n]-type receptors (SA 2.1).} 
These receptors bind a variety of drug-like molecules, some with high affinity.
\vspace{-0.2in}
}
\end{figure}

\textbf{Cucurbituril derivatives for host-guest binding.} 
%DLM: A bit odd to have the OA section start with a history header and this one not. May need to address.
Building on previous success with cucurbit[$n$]uril (abbreviated CB[$n$]) experiments for SAMPL challenges~\cite{ma_acyclic_2012-2, cao_absolute_2014, she_glycoluril-derived_2016}, we will conduct a series of new experiments on these receptors for five new challenges,
 with experimental work conducted by co-investigator Isaacs, an expert on these systems who provided data for previous SAMPL challenges.
CB[$n$] receptors are particularly well suited to our goals because they exhibit: (1) high binding affinities toward suitable guests in water comparable to protein-ligand affinities (routinely $\mu$M to nM; occasionally pM to fM)~\cite{cao_attomolar_2014, liu_cucurbituril_2005, mock_structure_1986, assaf_cucurbiturils:_2015, moghaddam_new_2011, shetty_can_2015, biedermann_release_2012}, (2) high selectivities between structurally related guests which translate into large $\Delta \Delta G$ values~\cite{isaacs_stimuli_2014}, (3) low molecular weights (1--2 kDa) permitting high levels of theory to be used, and (4) highly restricted conformational degrees of freedom, reducing conformational sampling challenges often seen in protein-ligand binding.
For SAMPL6-10, we will resynthesize a series of CB[$n$]-type receptors of increasing complexity, measure $K_a$ values, and determine host-guest stoichiometry and geometry toward pharmaceutically relevant guests (selected drugs) in order to stringently test methods for predicting binding.  
Figure~\ref{figure:CB} shows the chemical structures of three hosts---Me$_4$CB[8]~\cite{vinciguerra_synthesis_2015}, glycoluril hexamer~\cite{lucas_templated_2011}, and acyclic CB[$n$]-type receptors~\cite{ma_acyclic_2012, ma_acyclic_2012-1, zhang_acyclic_2014, gilberg_acyclic_2015, sigwalt_acyclic_2016, zhang_acyclic_2014-1} which span a range in terms of level of preorganization and formal charge.


\textbf{SAMPL6-10 cucurbituril challenges.} 
\emph{For SAMPL6}, we will measure $K_a$ and $\Delta H$ values, stoichiometry, and geometry for the interaction of Me$_4$CB[8] (a soluble CB[8] derivative) with 15 guests (selected top drugs, Table~\ref{table:CB}) by either direct or competition isothermal titration calorimetry (ITC), UV/Vis or fluorescence indicator displacement assay, or NMR competition experiments, as previously~\cite{cao_attomolar_2014, liu_cucurbituril_2005, ma_acyclic_2010, she_glycoluril-derived_2016}.  
Our selection of Me$_4$CB[8] binding to top drugs allows us to modulate the computational complexity by: 1) changing host flexibility (e.g. Me$_4$CB[8] can exhibit ellipsoidal deformation)~\cite{vinciguerra_synthesis_2015}, 2) allowing the possibility of binary or ternary (e.g. 1:1 and/or 1:2 host:guest) complexes~\cite{ko_supramolecular_2007, barrow_cucurbituril-based_2015, urbach_molecular_2011}, 3) using drugs with several potential binding epitopes or modes to induce sampling issues.  Host:guest stoichiometry and geometry (e.g., which binding epitope is complexed) will be addressed by ITC $n$ values, Job plots monitored by UV/Vis or NMR~\cite{connors_binding_1987}, and by $^1$H NMR complexation induced changes in chemical shifts~\cite{masson_cucurbituril_2012}.  
All studies will be conducted in phosphate buffered saline (p$H$ 7.4 with physiological salt) which introduces its own complexities due to salt competition for binding~\cite{marquez_mechanism_2004, mobley_predicting_2016}. 
\emph{SAMPL7} will revisit the same host, but use 15 different guests be selected from commercial sources on the basis of reference calculations (on a larger set of guests) to ensure that they cover substantial dynamic range and/or exhibit affinities that depend substantially on the force field or water model, thus effectively testing our force fields and methods.
For \emph{SAMPL8}, we will focus on binding of the same 15 drugs (Table~\ref{table:CB}), but to glycoluril hexamer. 
This host introduces the complication of increased conformational dynamics, and influences the number and energy of solvating (and unusually coordinated) water molecules implicated in the high binding constants for CB[$n$]-guest complexes~\cite{biedermann_release_2012, biedermann_hydrophobic_2014}.  
The selected drugs include several with p$K_a$ values in the 3.8 to 7.4 range; given that CB[$n$]-type receptors (like biomolecular receptors) can induce p$K_a$ shifts in their guests of up to 4 p$K_a$ units~\cite{saleh_activation_2008, nau_deep_2011, ghosh_strategic_2012}, this will test how well models can predict these effects. 
Additionally, it will couple nicely with the focus on p$K_a$ values in Aim 1.
\emph{SAMPL9} will revisit glycouril hexamer with the same 15 guests from \emph{SAMPL7}.
\emph{SAMPL10} will shift to acyclic CB[$n$]-type receptors (e.g. M1C$_3$, M1C$_0$, and M1PC$_0$ that contain anionic solubilizing groups attached via different linker lengths.  
As in SAMPL3~\cite{muddana_sampl3_2012}, these acyclic CB[$n$]-type receptors introduce conformational complexity, and water interactions play a key role.
Moreover, the presence of 4 anionic groups near the cavity will likely impact the balance between ion-dipole interactions and solvation of the free host.

\textbf{Subaim 2.2. Gibb deep cavity cavitands for host-guest studies} 

%Gibb science

\begin{figure}[h]
\begin{centering}
\resizebox{\textwidth}{!}{\includegraphics{figures/gdccs.pdf}}

\end{centering}

\vspace{-0.1in}
\caption{\footnotesize {\bf Gibb deep cavity cavitands for the SAMPL6-10 datasets (SA 2.2).} These hosts bind a variety of carboxylate and trimethylammonium guests in a strongly salt-dependent manner, providing a stringent test of our ability to model salt-dependent binding.
\label{figure:gdccs}
\vspace{-0.2in}
}
\end{figure}

{\bf History of GDCC SAMPL challenges.} During SAMPL4~\cite{gibb_binding_2013} and SAMPL5~\cite{sullivan_binding_2016} we focused on two specific GDCC hosts: the octa-acid 1 (R = H) and another octa-acid variant with four methyl groups at the portal of the binding pocket (1, R = Me). 
These studies used isothermal titration calorimetry (ITC) to measure the thermodynamics of (1) host {\bf 1} (R = H) binding a range of 9 carboxylate guests,
and (2) the binding of 6 carboxylate and trimethylammonium guests to both hosts ({\bf 1}, R = H and Me; Figure~\ref{figure:gdccs}).  
In both cases $^1$H-NMR titration was also used to confirm ITC-derived free energies of binding.  
As noted above, co-solvent effects and water rearrangements posed particular challenges for predicting binding in these hosts. 
SAMPL5 emphasized how differences in the shape of the hydrophobic pocket of the host can have a profound effect on affinity for some guests~\cite{yin_overview_2016}.
% Comment on value to community

{\bf Novel deep cavity hosts probe the effects of binding site charge constellations.} 
For future GDCC datasets, we will expand on the range of hosts by including {\bf 2} and {\bf 3} in our ITC studies (Figure~\ref{figure:gdccs}).  
Like cavitand {\bf 1}, host {\bf 2} is an octa-acid derivative.  However, the four benzoate groups are relocated from the extreme exterior in the case of {\bf 1}, to the rim of the binding pocket in {\bf 2}.  
We surmise that this will have a direct effect on the binding of charged guests as well as an indirect effect on guest complexation via changes to the solvation of the empty host.  
Octa-trimethylammonuim cavitand (``positand'' {\bf 3}) has the same overall architecture as host {\bf 1}, but inverts the charges on the water solubilizing exterior coat.  
While it is not yet clear if this switch in groups relatively remote from the pocket will directly affect guest complexation, results from related systems suggest it can (unpublished). 

{\bf SAMPL6-10 deep cavity cavitand datasets.} 
%Make sure we list # compounds for each (and ideally this would be increased!)
Data for SAMPL6 will focus on how well the effect of host carboxylate substituent location can be predicted, and will involve hosts {\bf 1} and {\bf 2} with a set of five previously uninvestigated guests.  
Guests will be selected from commercial sources on the basis of reference calculations in a similar manner to SAMPL7 in Subaim 2.1, specifically picking guests which have broad dynamic range and, here, have marked differences in affinities between hosts.
SAMPL7 will provide a second iteration of this experiment to test algorithmic improvements in predictive modeling following SAMPL6 by comparing hosts {\bf 1} and {\bf 3} with a different set of guests.  
We anticipate that because of the relative remoteness of the charged groups in these two hosts, the effects of switching charges will be subtler than the differences between {\bf 1} and {\bf 2}.  
SAMPL8 will consider the effect of common biologically-relevant counterions/salts on guest binding, comparing the effects of NaCl and NaI on the complexation of five guests to {\bf 1}.  
We have previously shown that iodide has a weak affinity for the binding pocket of {\bf 1}, while sodium ions have an affinity for the outer carboxylates~\cite{carnegie_anion_2014}, requiring modeling to capture the differential affinities of these ions in addition to guest affinities to successfully model the observed affinities.  
SAMPL9 will follow up on this by examining the effects of these same two salts on the complexation of five guests to {\bf 3}, again giving the modeling community time to incorporate algorithmic improvements following SAMPL8. 
While we have not yet quantified salt affinities to host {\bf 3}, we expect the iodide to have affinity for both the pocket and the positively charged solubilizing groups.  
For SAMPL10 we will consider the effects of co-solvents on the binding of five guests to {\bf 1} and {\bf 2} to probe the effect of co-solvent competition for the binding site, as well as effects co-solvents may have in weakening the hydrophobic effect. 
While the number of guests considered in each challenge is relatively small, the total number of binding affinities measured is significant across the full family of hosts, meaning that the full data set will be of considerable value as a benchmark set~\cite{mobley_predicting_2016}. 


\textbf{\underline{Aim 3: Develop model protein-ligand systems that isolate specific modeling challenges of drug targets.}}

%Cast in terms of crowdsourcing again. D3R can't measure progress because target data is different each time and different methods do well. We want clear way to measure progress, so we want to go back to same systems again and again... 
We seek to drive advances in quantitative modeling of protein-ligand interactions.
While D3R~\cite{Gathiaka:2016:JComputAidedMolDes} benchmarks accuracy for targets of pharmaceutical interest, it does not provide a clear route to improving poor performance because the large number complexities exhibited by these targets make it difficult to identify clear points of failure~\cite{ignjatovic_binding-affinity_2016, deng_large_2016, sunseri_d3r_2016, Gathiaka:2016:JComputAidedMolDes}.
For example, while kinases are targets of great interest to drug discovery, blind challenges involving kinase targets conflate issues of slow protein conformational dynamics~\cite{Lin:2013:Proc.Natl.Acad.Sci.}, protonation state effects of both protein~\cite{Shan:2009:PNAS} and ligand~\cite{Szakacs:2005:JournalofMedicinalChemistry,Grante:2014:SpectrochimicaActaPartA:MolecularandBiomolecularSpectroscopy}, charged ligands, and the modeling of complex divalent salt environments and phosphorylation state effects along with the standard challenges of conformational sampling and forcefield accuracy.
Thus D3R exercises serve the community well to understand current accuracy, but {\bf blind challenges on complex pharmaceutical targets have limited ability to rapidly advance quantitative predictive modeling.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% FIGURE: HSA SAMPL6 CHALLENGE
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[h]
\begin{centering}
\resizebox{\textwidth}{!}{\includegraphics{figures/hsa-figure-draft7.pdf}}

\end{centering}
\vspace{-0.1in}
%Update numbers.
\caption{\footnotesize {\bf The SAMPL6/7 protein-ligand challenge focuses on soluble drug fragment binding to human serum albumin (HSA) (Aim 3).}
\emph{(A)}~SAMPL6 will study binding of a library of 96 small soluble druglike fragments to recombinant HSA, with an engineered HSA mutant used for SAMPL7.
\emph{(B)}~HSA has at least eight known binding sites, with two major well-characterized sites (green, Sudlow's Site I; purple, Site II) that bind a variety of drugs (figure from~\cite{Hall:2013:JournalofChemicalInformationandModeling}).
Two fluorescent probes---dansylamide and dansylglycine---bind with $\sim$$\mu$M affinity and high selectivity to Site I and Site II, respectively; both exhibit binding-enhanced fluorescence at 480 nm, and can be used to site-specifically probe ligand affinities by competition.
\emph{(C, D)}~Binding affinities of soluble molecules can be measured by isothermal titration calorimetry (ITC); here, we show the \emph{(C)} differential power and \emph{(D)} integrated injection heats for the ITC titration of HSA by naproxen sodium collected using the Chodera lab automation pipeline;
\emph{(E)}~HSA tryptophan fluorescence quenching can also be used to measure ligand binding affinity; here, HSA titration by diclofenac is shown, with the inset plot showing percent quenching at 346 nm~\cite{Epps:1999:JournalofPharmacyandPharmacology,Bou-Abdallah:2016:TheJournalofChemicalThermodynamics}.
\emph{(F)}~Direct fluorescence binding assay of Dansylamide (fluorescent ligand) and HSA collected on the Chodera lab automation system. 
The binding curve can be constructed based on binding-induced fluorescence emission at 480 nm.
\vspace{-0.25in}
\label{figure:hsa-challenge}}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\eject

{\bf We take the alternative approach of identifying and developing specific protein-ligand systems which \emph{isolate} individual accuracy-limiting effects in a series of prospective challenges.}
By developing \emph{model binding systems}---real protein-ligand systems that may be of pharmacological interest, but comprised of single-domain proteins binding to a simple ligand series free of complex phenomena---we can study systems of complexity intermediate between completely artificial systems (such as the T4 lysozyme L99A system developed by Shoichet and Matthews~\cite{mobley_predicting_2007,merski_homologous_2015, mobley_predicting_2016}) and complex pharmaceutical targets where multiple modeling challenges make it difficult to learn from failure (Figure~\ref{figure:mining-for-model-systems}A).
This process focuses the field on identifying and evaluating multiple solutions to selected accuracy-limiting effects (such as how to deal with ligand and protein protonation-state issues~\cite{Onufriev:2013:QuarterlyReviewsofBiophysics}, slow protein conformational dynamics, etc.) while avoiding other complicating factors.

While model systems have had ample success in driving progress in individual research laboratories, community participation in blind challenges amplifies their power.
For example, SAMPL3 featured the binding of small, rigid charged molecules to bovine trypsin~\cite{Newman:2011:JComputAidedMolDes}, and rapidly focused the field on the deficiencies of current alchemical free energy methodologies in treating the binding of charged ligands.
Within two years, multiple laboratories had developed practical solutions to effectively handle charged ligand binding~\cite{Rocklin:2013:TheJournalofChemicalPhysics, lin_overview_2014, Reif:2014:JournalofComputationalChemistry}.

\textbf{SAMPL6-10 model protein-ligand challenges.} 
We will introduce a new model protein-ligand system each year (revisiting the prior year's system if this becomes too difficult), with multiple challenges on each system (Figure~\ref{figure:timeline}) to allow iterative improvement and assessment.
Our SAMPL6 data will focus on binding of small soluble drug fragments to one particular protein (below). 
However, maximizing gains in this area requires adapting subsequent challenges based on deficiencies identified by previous D3R/SAMPL challenges.
Therefore, subsequent model systems will be rapidly identified and developed using our new informatics platform (below).

\textbf{SAMPL6: Assessing predictive modeling of binding to multiple weak sites via measuring fragment binding to human serum albumin (HSA).}
HSA, the most abundant blood plasma protein, has a remarkable ability to bind a great variety of small molecule drugs in multiple binding sites (Figure~\ref{figure:hsa-challenge}B)~\cite{Fasano:2005:IUBMBLife(InternationalUnionofBiochemistryandMolecularBiology:Life)}.
As a result, HSA not only helps isolate the challenge of multiple weak ligands binding to a stable rigid protein, but it is also a pharmacologically relevant because of how it drastically modulates drug pharmacokinetics~\cite{Hall:2013:JournalofChemicalInformationandModeling}.
HSA has at least \emph{eight} known binding sites, with numerous crystal structures available for drugs binding to two predominant sites (Site I and II)~\cite{Hall:2013:JournalofChemicalInformationandModeling}.
Small soluble molecules resembling drug fragments are highly likely to bind to HSA ($\ge$90\% of such fragments, as detected by SPR~\cite{Elinder:2011:JournalofBiomolecularScreening}), providing an experimentally-tractable diverse set of ligands spanning several orders of magnitude in affinity~\cite{Elinder:2011:JournalofBiomolecularScreening}.
As current advanced methodologies such as alchemical free energy calculations currently assume a single well-defined binding site with high affinity~\cite{Gilson:1997:BiophysicalJournal}, this dataset will allow the isolation of the effect of weak multiple binding from the majority of other confounding factors in protein-ligand binding.
As HSA is relatively rigid, and computational methods already show some promise in computing binding affinities to HSA~\cite{Hall:2013:JournalofChemicalInformationandModeling,Lexa:2014:PLoSONE,Evoli:2016:bioRxiv}, this is an optimal model system for SAMPL6.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{wrapfigure}[37]{r}[0pt]{9.5cm}
\vspace{-0.25in}
\begin{centering}
\resizebox{9.0cm}{!}{\includegraphics{figures/model-systems-figure-draft4.pdf}}
\vspace{-0.15in}
\end{centering}
\footnotesize
\caption{\label{figure:mining-for-model-systems}  
\textbf{Mining model protein-ligand systems to focus on individual modeling challenges via a structural and chemical informatics platform (Aim 3).}
We are developing a structural and chemical informatics system called TargetExplorer [\url{https://github.com/choderalab/targetexplorer}] that applies successive filters to all potential protein-ligand systems for which structural data is available. 
Suitable model systems should meet all experimental tractability criteria (green box) and possess only a few challenging properties, ideally only one (blue box).
Tractability of experimental affinity measurements includes properties like known ligands with potentially fluorescent scaffolds (for fluorescence competition assays), highly soluble ligands (for ITC), or ligands above a minimal mass (for SPR or MST).
Additional filters annotate experimentally tractable systems with their potential computational challenges, including charged ligands or potential ligand protonation state or tautomer effects~\cite{Martin:2009:JournalofComputer-AidedMolecularDesign} (deduced from predicted aqueous protonation/tautomer energies); potential protein protonation state effects (deduced from MCCE2 calculations~\cite{Song:2009:JournalofComputationalChemistry}); protein conformational changes (deduced from variation in protein conformation or the presence of unresolved loops in protein-ligand crystal structures); the presence of post-translational modifications that may affect affinity (deduced from Uniprot annotations); coordinated metals (identified in crystal structures); and ordered waters (present in multiple crystal structures).
}
\end{wrapfigure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Recombinant HSA will be expressed in \emph{E.~coli} and purified via refolding from inclusion bodies~\cite{Latta:1987:Bio/Technology}, then defatted at low pH~\cite{Lang:2015:BiotechnologyProgress} to ensure the resulting protein is free of the glycosylation and bound fatty acids found in plasma-isolated HSA~\cite{Lang:2015:BiotechnologyProgress}.
Recombinant expression will also allow a mutant form of HSA (engineered via single-primer mutagenesis) to be fielded for SAMPL7 (Figure~\ref{figure:hsa-challenge}).
We will obtain a diverse library of 96 soluble drug-fragment-like molecules in pre-plated format as dry compound, and use our automated isothermal titration calorimetry (ITC) pipeline (Figure~\ref{figure:hsa-challenge}C,D) to characterize overall binding affinities to HSA.
The same ligands pre-plated in DMSO format will be used to conduct a separate set of fluorescence titration assays (monitoring tryptophan fluorescence quenching, Figure~\ref{figure:hsa-challenge}D) and competition assays with site-specific fluorescent probes (Figure~\ref{figure:hsa-challenge}B) to resolve site-specific affinities to Sites I and II.
We will field several levels of challenges, including challenges focused on affinities to Sites I and II, as well as challenges focused on predicting overall affinity and stoichiometry.
%Remove redundancy between this paragraph and the caption to save space? (i.e. refer to there for some of this content?)

\textbf{SAMPL7-10: Rapid development of new tailored model systems using a novel informatics platform.} We are developing a novel informatics platform aimed at identifying protein targets that can be rapidly developed into experimentally- and computationally-tractable model systems focusing on individual challenges (Figure~\ref{figure:mining-for-model-systems}).
This tool filters all known protein targets with structural data available in the PDB, first selecting for experimental tractability, then annotating experimentally tractable targets to determine which targets possess (or are likely to be free of) specific challenges for physical modeling.
This will allow us to select systems which introduce only specific modeling challenges.

The Chodera lab has developed an automated wetlab to facilitate the development of such model protein-ligand systems using bacterial expression (see Equipment and Facilities).
Potential targets matching desired challenge criteria will be screened for bacterial expression using high-throughput cloning, transformation, and expression testing, with purity and yield assessed by capillary electrophoresis on a Caliper GXII.
Targets will be screened for stability in various buffers using Thermofluor thermal shift assays~\cite{Reinhard:2013:ActaCrystallographicaSectionFStructuralBiologyandCrystallizationCommunications}.
Ligands identified via TargetExplorer as spanning a wide dynamic range of binding affinities will be purchased as dry powder stocks and prepared for assay by highly accurate gravimetric solution preparation techniques using a Quantos automated balance.
Our lab has access to a wide variety of biophysical techniques for quantitative measurement of protein-ligand binding affinities, including fluorescence (if fluorescent probe ligands are available), absorption (e.g.~Soret band shifts), automated isothermal titration calorimetry (provided ligands are sufficiently soluble), surface plasmon resonance, microscale thermophoresis (MST), luminescence, and alphascreen; all except MST are fully automated. 

We take a twofold approach to developing challenge datasets:
First, we will purchase and assay small molecules similar to known ligands, presuming that these molecules are likely to have measurable affinities.
Second, using single-primer quick-change mutagenesis, we will introduce site-directed mutants to modulate the binding affinities of known ligands.
This can be performed and screened for expression in 96-well format.
Thus, datasets will consist of a matrix of protein mutants and ligands, providing opportunity to deeply explore the effects of interest.

%Retitle aim? Get crowdsourcing in it? 
%Don't start with datasets. THIS AIM IS THE MAIN POINT. The datasets are peripheral. Refocus.
%Simplify language slightly: "New, progressive, targeted nature of the data..." too wandering.
%"...subsequent data release activities" awkward, it's just people using the data.
%I'll need to recap in this section the coupling between Aim 1 data and Aim 2-3 data.
\textbf{\underline{Aim 4: Field community blind challenges to advance quantitative biomolecular design.}}
The value of the targeted datasets generated in Aims 1--3 will be amplified enormously by the strategic release of this data through iterative, coordinated SAMPL blind challenges (Figure~\ref{figure:timeline}).
These blind challenges are designed to test the state of the art, provoke new methodological and force field innovations, allow comparative evaluation of methods, and drive downstream improvements.
The new, progressive, targeted nature of the data generated means that SAMPL challenges will now build on one another, and for success in later challenges, participants must build on lessons learned from prior challenges.
SAMPL challenges and subsequent data release activities will therefore facilitate rapid cycles of application, learning, and improvement.
Each iteration will likely yield its own incremental benefits (e.g.. as in Figure~\ref{figure:sampl5_logD}) for molecular design, in addition to contributing to progress towards our larger goals.

{\bf SAMPL blind challenges.} Challenges will have yearly submission deadlines and involve roughly the same size data sets as prior challenges.
The full timeline for SAMPL challenges (Figure~\ref{figure:timeline}) will be made available on the website [\url{https://drugdesigndata.org/about/sampl}] at the outset, allowing participants to plan their work and select what challenges to be involved in.
As experimental data for each component becomes available and is curated, input files and challenge details will be made available at least six months prior to the challenge deadline; data not yet available at that time will be held for a subsequent challenge (with the exception of three months for year 1 due to startup timescales).
%reference survey results?
As in prior SAMPLs, submissions will be handled by a web upload service on the SAMPL website (which will be migrated to separate hosting if the D3R effort is not renewed) which validates submissions to ensure that they meet format standards we specify along with the challenge details. 
As in SAMPL4 and SAMPL5, analysis will also be conducted by our automated Python framework, and results returned automatically online.
All participant submissions and methodology descriptions will (as before) be made available publicly on the website, along with participant information (except for participants who specifically request to remain anonymous prior to submission).
Aggregate statistics and historical performance will also be made available on our website, along with a record linking publications to historical submissions.
%Shift towards SUBMISSION OF ALGORITHMS and running on the cloud? (e.g. Orion?) "A stretch goal". Would also be good for student training! Participants could submit their methods and a budget...  (See paragraph at end of this section).

\textbf{Our goal is not just to run blind challenges, but to advance modeling by helping participants identify both modeling failures and potential solutions.} 
%Seems like we also have to make it easy for people to exchange knowledge/repeat calculations, etc. 
To achieve this, we provide guidance to participants as to what known modeling issues we expect may be relevant when providing details on each SAMPL component.
For a host-guest system, for example, we might highlight known buffer/salt effects, protonation state challenges, and point out previous work on sampling challenges, with pointers to the relevant experimental work and to modeling work from past SAMPL challenges and elsewhere~\cite{mobley_predicting_2016}.
This helps participants design their approach.
{\bf Additionally, we will run reference calculations using current best practices.} 
This serves several purposes:
It provides a test of the current methods and force fields we select; 
it helps facilitate learning---we announce what calculations we plan to perform, make input files available in a wide variety of formats~\cite{shirts_lessons_2016, yin_overview_2016, Bannan:2016:JComputAidedMolDes}, and others can repeat our calculations with a different method but same system and force field to compare methods, or swap force field but keep the method and system fixed to compare force fields, etc.; 
and it allows us to conduct sensitivity analysis, as by varying the conditions of our simulations (protonation state, tautomer, etc.,~\cite{Bannan:2016:JComputAidedMolDes}) we can see how much this impacts calculated values and thereby how important it is, even if participants don't do these tests.
Reference calculations have, for example, helped us highlight the importance of a small amount of water in cyclohexane for accurately calculating log~D values, show how an incorrect tautomer could affect calculated values by many log units~\cite{Bannan:2016:JComputAidedMolDes}, and discover that small forcefield modifications could significantly improve results on hydration free energies~\cite{mobley_blind_2014-1}.
To further aid follow-up studies, we will make the input files, results, and simulation workflows used for our reference calculations---along with the data---available via GitHub and Docker Hub.


Physical methods are only valuable if they can reliably outperform alternate methods, so \textbf{a new focus of SAMPL6-10 will be selecting quality null models and running them to provide a point of comparison for participants}, going far beyond previous SAMPL nulls~\cite{Bannan:2016:JComputAidedMolDes, paranahewage_predicting_2016, muddana_sampl4_2014}).
%Mehtap wondered what null models are. Could be this is an additional argument to strike this paragraph. Would that give me room to make better headers/reformat?

Following submission and analysis of each SAMPL challenge, challenge results will be released and discussed, with SAMPL workshops allowing more formal presentations on and discussion of results in years 1, 3, and 5. 
Workshops will run every two years at the request of past participants, and will be co-run and co-hosted with D3R Grand Challenge workshops (see support letter).
During the off years, SAMPL challenges will still run, but discussion of and dissemination of results will be via asynchronous means (as discussed below) and a ``virtual workshop'' consisting of talks and interaction over Google Hangouts or YouTube Live.
While coordination with D3R will mostly be at the level of workshops, we will also ensure that SAMPL challenge submission deadlines are offset from D3R deadlines to allow maximum community participation in both efforts.
If the D3R effort is not renewed beyond its current funding cycle, we will run SAMPL workshops independently, controlling costs via the model we use for our Workshops on Free Energy Methods in Drug Design---specifically, most participants will pay their own way to the workshop, and we will seek pharmaceutical and software industry sponsorships to defray costs. 

{\bf Dissemination of results and data.}
Rapid dissemination of results is critical so that new insights can be used in subsequent challenges.
We will continue to publish special issues of the \emph{Journal of Computer Aided Molecular Design} (JCAMD) collecting publications related to each year's SAMPL challenges (see Letter of Support).
To ensure immediate availability of reports, we will strongly encourage prepublication sharing of results and analysis, including both slides and posters from SAMPL meetings (via F1000 Research) and paper preprints (via bioRxiv).
We also want to ensure that participants learn from one another by rapid exchange of ideas outside of formal workshops and meetings.
While this has happened in the past---for example, when participants using similar methods work together after the SAMPL meeting to identify the origin of these discrepancies~\cite{monroe_converging_2014, yin_overview_2016, bhakat_resolving_2016, bosisio_blinded_2016, mobley_predicting_2016}---we hope to accelerate this kind of collaboration.
To facilitate more open communication between the community, we will use collaboration software---such as Slack, which facilitates scientific communication for the NASA/JPL Mars Rover teams and NSF antarctic scientific research teams---to build a community discussion platform, 
facilitating a process of learning from one another more rapidly than normal publication channels.

%Get a post-challenge analysis phase (see DREAM stuff) in here

%Shift emphasis and header in this paragraph. The data IS valuable and that's worth noting. But maybe we should shift to making sure top performing methods are available to everyone if at all possible, and key take-aways are disseminated for use for the next challenge? 
\textbf{Each dataset will have a life cycle of collection, curation, blind challenges, and public dissemination}.
In the past, the unfunded nature of SAMPL has forced us to primarily emphasize the \emph{blind challenges} and pre-challenge \emph{curation} aspects, with isolated forays into collection~\cite{rustenburg_measuring_2016, Newman:2011:JComputAidedMolDes}. 
This work now considers the full life cycle, with Aims 1--3 dealing primarily with collection and pre-challenge curation.
Post-challenge, 
datasets will receive additional curation, then be released as standard test or benchmark sets that allow retrospective evaluation of methodologies on high-quality data~\cite{mobley_predicting_2016}. 
The FreeSolv dataset, for example, includes a large number of calculated and experimental hydration free energies from SAMPL0--4, and provides a standard benchmark dataset for hydration free energy calculations~\cite{Mobley:2014:JComputAidedMolDes}. %add new ref too
Post-challenge curation will receive new attention here; in the past, lack of resources has always prevented follow-up experimental work, even when the data clearly indicated it was warranted (such as the puzzling issues with dynamic range for log~D values in SAMPL5~\cite{rustenburg_measuring_2016, Bannan:2016:JComputAidedMolDes}).
The requested budget will allow follow up experiments motivated by computation when warranted.
Dissemination is the final stage in the data life cycle (see Resource Sharing Plan); we will make the data (including primary data, processed data, and our analysis of challenge submissions) available freely and publicly with permanent, citeable DOIs; ensure relevant data is deposited in standard community repositories (e.g.~BindingDB~\cite{Liu:2007:Nucl.AcidsRes.}); and guarantee data longevity via backup hosting on library archival facilities (such as the UC's DASH (\url{https://dash.cdlib.org/})).

	%   Methodology archival? Push for containerization of tools, maybe in conjunction with other known efforts. [Currently short on space, but putting something very short on this here anyway.]
We will also push for containerization of tools and methods in conjunction with other efforts such as AutoDesk's Molecular Design Toolkit and the NSF Molecular Sciences (MolSci) initiative. %Fix acronym
Our vision is that long-term, instead of participants submitting a set of predictions, they would also submit the entire workflow they applied via Docker containers allowing reproducibility and repurposing, ensuring dissemination workflows, not just results.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% COLLABORATION MANAGEMENT PLAN
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%{\large \bf COLLABORATION MANAGEMENT PLAN} % 1 paragraph
{\Large \bf COLLABORATION MANAGEMENT PLAN}

We have a strong previous history of successful collaboration, with Mobley and Chodera having co-authored roughly a dozen publications and organized several workshops and other initiatives.
Mobley, Isaacs, and Gibb have also worked together to coordinate past SAMPL challenges, and Mobley and Gibb a previous NSF workshop. 
PI Mobley will oversee the project, with teams for the other aims (Aim~1: Mobley \& Chodera; Aim~2.1: Isaacs; Aim~2.2: Gibb; Aim~3: Chodera; Aim~4: Mobley \& Chodera) involving the other co-investigators as needed.
Meetings will consist of a monthly Google Hangout and a yearly in-person planning meeting. %more frequent as needed
Chodera and Mobley will communicate more frequently due to the interlinked nature of their work.
Publications are expected to be largely dictated by the overall Timeline, with an experimental publication associated with each challenge component being prepared for distribution to participants along with their results.
Conflict resolution is expected to be straightforward, but if any serious difficulties arise, Michael Gilson (UCSD) will arbitrate given our close connections with D3R.

{\Large \bf OUTLOOK}

Physical methods have been slow to achieve their promise in binding prediction, in part because truly significant innovations are so hard to recognize due to a lack of standard tests and benchmarks, and in part because of an ``applications first'' approach which seems to plague our community where we rush to apply our methods to problems of pharmaceutical relevance without ensuring they can tackle simpler, better-understood problems first.
%Rephrase - crowdsourcing. We're driving innovations in field. Downplay data, play up challenges.
Here, we propose an innovative extension of the successful series of SAMPL blind challenges, generating novel experimental data to drive improvement of the methods in our field and help them become pharmaceutically relevant -- beginning with relatively simple physical property prediction and progressing to challenging problems in biomolecular recognition via a series of carefully designed intermediate steps.
SAMPL already has a strong track record of success, and funding will ensure dramatically increased impact and continued success.
The proposed series of carefully tailored challenges will focus our community on a variety of problems which we \emph{can} realistically resolve in the near term, resulting in dramatic improvements in computational molecular design.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% BIBLIOGRAPHY
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\eject

%\footnotesize
%\scriptsize
%\bibliographystyle{acm}
\bibliographystylesampl{nci}
\bibliographystyle{nci}
%\bibliographystyle{nar}
%\bibliography{full}{sampl-r01}{BIBLIOGRAPHY AND REFERENCES CITED}
%\bibliography{sampl}{sampl-r01}{REFERENCES FROM PRIOR SAMPL CHALLENGES}

\bibliographysampl{sampl}
\eject
\bibliography{sampl-r01}

\end{document}